tools/c7n_azure/c7n_azure/query.py

```diff
@@ -136,6 +141,11 @@ def __new__(cls, name, parents, attrs):
 @six.add_metaclass(QueryMeta)
 class QueryResourceManager(ResourceManager):
 
+    class resource_type(object):
+        service = ''
```

Comment:
better to reference a base metadata class here (defined outside the class closure) if you want to defaults, see gcp and k8s providers for examples of the same.